app/utils.ts

```diff
@@ -25,6 +25,26 @@ export function safeRedirect(
   return to;
 }
 
+/**
+ * If the request's url contains "www.", return the non-www version of the url,
+ * and retain all protocol, path & query info.
+ *
+ * A trailing slash will be added to a hostname if missing the path.
+ *
+ * Otherwise, return null.
+ */
+export const redirectToNonWww = (requestUrl: string): string | null => {
```

Comment:
j/ a suggestion, but the name of this function implies that calling it will redirect the page. maybe `getRedirectUrlIfWww` or something like that? ([naming is hard](http://thecodelesscode.com/case/220))

Reply:
Fixed, thanks!